partials/Frontend/Frontend.tsx

```diff
@@ -0,0 +1,14 @@
+import HeroBlockTechnology from '../../components/HeroBlockTechnology/HeroBlockTechnology';
+
+// TODO add data from json
```

Comment:
are we sure we need this component?
can we use the HeroBlockTechnology component in the index.tsx of the page?